spock-junit4/src/test/groovy/org/spockframework/smoke/groovy/UsageOfNotYetImplemented.groovy

```diff
@@ -16,28 +16,33 @@
 package org.spockframework.smoke.groovy
 
 import groovy.transform.NotYetImplemented
-
 import junit.framework.AssertionFailedError
-
 import spock.lang.Specification
 import spock.lang.FailsWith
 
-// make sure that Groovy's @NotYetImplemented works with Spock
+//TODO: Handle also new groovy.transform.NotYetImplemented in Groovy 3 - https://github.com/spockframework/spock/issues/1127
```

Comment:
Unrelated junit4 removal, I will cover it (the new transformation) in a separate PR.